projects/1.21/assets/the-undergarden/undergarden/lang/zh_cn.json

```
@@ -263,24 +315,35 @@
   "block.undergarden.wigglewood_stairs": "摇蜿木楼梯",
   "block.undergarden.wigglewood_trapdoor": "摇蜿木活板门",
   "block.undergarden.wigglewood_wood": "摇蜿木",
+  "commands.undergarden.infection.cannot_infect": "%s免疫感染",

```

```suggestion
  "commands.undergarden.infection.cannot_infect": "%s对感染免疫",
```
这个应该是聊天栏提示吧，`XX免疫感染`感觉稍微有点怪